JSDemos/Demos/TreeList/CellEditing/description.md

```diff
@@ -1 +1,7 @@
-This example demonstrates editing in _&laquo;cell&raquo;_ mode. A&nbsp;click on&nbsp;any cell switches it&nbsp;into the editing state. Once you click outside the cell, the TreeList saves your changes. Note that several columns are made uneditable by&nbsp;setting their **allowEditing** property to _false_.
\ No newline at end of file
+Users can modify TreeList data cell by cell. In this mode, only one cell can be in the edit state at a time. TreeList saves changes immediately after the focus leaves cell.
+
+To enable the cell mode, do the following:
+- Set the **editing**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) to *"cell*".
+- Assign **true** to the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties.
+
+When a user presses the button to add a new row, the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) handler populates the new row editor with data. In this demo, this function preselects *Not Started* in the `Status` drop-down field and presets the current date for the `Start Date` and `Due Date` fields.
```

Comment:
> this function preselects *Not Started* in the `Status` drop-down field

This is not a drop-down field. The drop-down field is on the screenshot: 
![image](https://user-images.githubusercontent.com/33040934/114396701-826d2700-9ba6-11eb-9263-5fb44f0789e2.png)

Also I still don't get what is a 'row editor'